install/local/addQTIExamples.php

```diff
@@ -15,18 +17,26 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2013-2014 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ * Copyright (c) 2013-2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
  *
  */
 
-use oat\tao\model\TaoOntology;
+use oat\oatbox\service\ServiceManager;
 
-$testClass = taoQtiTest_models_classes_QtiTestService::singleton()->getRootClass();
-$file = dirname(__FILE__) . DIRECTORY_SEPARATOR . 'qtiv2p1Examples.zip';
+$testService = ServiceManager::getServiceManager()->get(taoQtiTest_models_classes_QtiTestService::class);
+$testClass = $testService->getRootClass();
+$samplesDirectory = new DirectoryIterator(__DIR__);
 
 try {
-    $report = taoQtiTest_models_classes_QtiTestService::singleton()->importMultipleTests(new core_kernel_classes_Class(TaoOntology::TEST_CLASS_URI), $file);
-} catch (Exception $e) {
-    common_Logger::e('An error occured while importing QTI Test Example. The system reported the following error: ' . $e->getMessage());
+    foreach ($samplesDirectory as $file) {
+        if ($file->isReadable() && $file->isFile() && 'zip' === $file->getExtension()) {
+            $report = $testService->importMultipleTests($testClass, $file->getRealPath());
```

Comment:
I am getting this error: 

```
Fatal error: Uncaught Error: Cannot pass parameter 7 by reference in /var/www/html/taoQtiTest/models/classes/class.QtiTestService.php:583
Stack trace:
#0 /var/www/html/taoQtiTest/models/classes/class.QtiTestService.php(372): taoQtiTest_models_classes_QtiTestService->importTest(Object(core_kernel_classes_Class), Object(taoQtiTest_models_classes_QtiResource), Object(taoQtiTest_models_classes_ManifestParser), '/tmp/tmp2013265...', Array)
#1 /var/www/html/taoQtiTest/install/local/addQTIExamples.php(33): taoQtiTest_models_classes_QtiTestService->importMultipleTests(Object(core_kernel_classes_Class), '/var/www/html/t...')
#2 /var/www/html/taoQtiTest/scripts/update/Updater.php(1963): require_once('/var/www/html/t...')
#3 /var/www/html/generis/common/ext/class.UpdateExtensions.php(118): oat\taoQtiTest\scripts\update\Updater->update('38.5.0')
#4 /var/www/html/generis/common/ext/class.UpdateExtensions.php(65): common_ext_UpdateExtensions->updateExtension(Object(common_ext_Extension))
#5 /var/www/html/tao/models/classes/extension/Up in /var/www/html/taoQtiTest/models/classes/class.QtiTestService.php on line 583
```

In `taoQtiTest_models_classes_QtiTestService` line 583 we must set the 7th parameter to a variable before.

Reply:
I had incompatible version. Ignore it.